README.md

```diff
@@ -13,7 +13,7 @@ Can't wait to generate some fake data ? Follow the steps:
   
 ```elixir
 def deps do
-  [{:faker_elixir_octopus, "> 0.0.0",  only: [:dev, :test]}]
+  [{:faker_elixir_octopus, "~>0.13.1",  only: [:dev, :test]}]
```

Comment:
I thought this package was actually on version 0.0.0 and so I almost left the page without checking it out (I thought it would be new and so lack lots of features). I will revert this later.